src/psyclone/tests/psyir/nodes/structure_reference_test.py

```diff
@@ -129,8 +129,8 @@ def test_struc_ref_create_errors():
     with pytest.raises(TypeError) as err:
         _ = nodes.StructureReference.create(
             symbols.DataSymbol("fake", symbols.INTEGER_TYPE), [])
-    assert ("symbol that is (or could be) a structure, however symbol "
-            "'fake' has type 'Scalar" in str(err.value))
+    assert ("A StructureReference must refer to a symbol that is (or could be)"
+            " a structure, has been given a 'Scalar' with name: 'unknown'")
```

Comment:
"..., but has been given a ..."

Reply:
This is the right message (but was also missing the "in" operator), let me know if the current error string is ok